webcompat/templates/issue.html

```diff
@@ -20,7 +20,7 @@
     </article>
 </section> -->
 
-    <section id="issue" class="grid-row js-Issue">
+    <section id="issue" class="grid-row js-Issue" style="display: none">
```

Comment:
that's great! :)
could we change the inline style to the class `is-hidden`? I've tested it out and it works. 
Otherwise thank you for figuring it out so quickly! <3